guides/common/modules/proc_disabling-cockpit-on-project.adoc

```diff
@@ -17,7 +17,7 @@ This removes the *Web Console* button from the {ProjectWebUI}.
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {package-remove} rubygem-foreman_remote_execution-cockpit
+# {client-package-remove-el8} rubygem-foreman_remote_execution-cockpit
```

Comment:
Why a `client-*` command on ProjectServer?

Reply:
It was a client attribute before. I think this will be reworked with https://github.com/theforeman/foreman-documentation/pull/2660 anyway. Fixed for now.